spring-integration-core/src/main/java/org/springframework/integration/gateway/MessagingGatewaySupport.java

```diff
@@ -716,7 +723,7 @@ protected final ErrorMessage buildErrorMessage(@Nullable Message<?> requestMessa
 	 * @return the attributes.
 	 * @since 4.3.10
 	 */
-	protected AttributeAccessor getErrorMessageAttributes(Message<?> message) {
+	protected AttributeAccessor getErrorMessageAttributes(@Nullable Message<?> message) {
```

Comment:
Maybe it's time to add a `@NonNullApi` into this package?

Reply:
I'd rather not add new ones until current smells are cleaned. That said, that fact that this was flagged leads me to believe it is inherited from the parent package.